packages/hdwallet-ledger-webusb/src/transport.ts

```diff
@@ -91,6 +102,9 @@ export class LedgerWebUsbTransport extends LedgerTransport {
       }))
       if (coin) {
         response = await new (translateCoin(coin))(this.transport)[method](...args)
+      } else if (method === 'openApp') {
+        const [ appName ] = args
+        response = await openApp(this.transport, appName)
       } else {
         response = await (translateMethod(method))(this.transport)
```

Comment:
Can we just use the else and spread the args if there are any?

Reply:
Had to add a // @ts-ignore